src/momento/internal/aio/_vector_index_control_client.py

```diff
@@ -37,12 +38,35 @@ def __init__(self, configuration: VectorIndexConfiguration, credential_provider:
     def endpoint(self) -> str:
         return self._endpoint
 
-    async def create_index(self, index_name: str, num_dimensions: int) -> CreateIndexResponse:
+    async def create_index(
+        self, index_name: str, num_dimensions: int, similarity_metric: SimilarityMetric
+    ) -> CreateIndexResponse:
         try:
             self._logger.info(f"Creating index with name: {index_name}")
             _validate_index_name(index_name)
             _validate_num_dimensions(num_dimensions)
-            request = ctrl_pb._CreateIndexRequest(index_name=index_name, num_dimensions=num_dimensions)
+
+            if similarity_metric == SimilarityMetric.EUCLIDEAN_SIMILARITY:
+                request = ctrl_pb._CreateIndexRequest(
+                    index_name=index_name,
+                    num_dimensions=num_dimensions,
+                    euclidean_similarity=ctrl_pb._CreateIndexRequest._EuclideanSimilarity(),
+                )
+            elif similarity_metric == SimilarityMetric.INNER_PRODUCT:
+                print("here")
```

Comment:
Is this an errant debug statement?

Reply:
Good catch

Reply:
Fixed in https://github.com/momentohq/client-sdk-python/pull/383/commits/18d537f433286c406334ca77fa457407904e9a56